### PR TITLE
Respect classification of calendar events in activity stream

### PR DESCRIPTION
**Pull request by nickvergessen**
Based on #13331 by @fdellwing

### CODE REVIEW

**Comment by fdellwing** [comment]
Well, it's doing a lot more than my PR, but seems good :+1:

**Review by rullzer — APPROVED** [review]
Looks good. Quick smoke test didn't do :boom:

**Review by tcitworld — APPROVED** [review]

File: apps/dav/lib/CalDAV/Activity/Backend.php [hunk 1]
```diff
@@ -446,7 +453,8 @@ public function onTouchCalendarObject($action, array $calendarData, array $share
 						],
 						'object' => [
```
Comment (tcitworld) [inline apps/dav/lib/CalDAV/Activity/Backend.php]: How is this used afterwards ? Shouldn't it be translated ?

File: apps/dav/lib/CalDAV/Activity/Backend.php [hunk 2]
```diff
@@ -446,7 +453,8 @@ public function onTouchCalendarObject($action, array $calendarData, array $share
...
 							'id' => $object['id'],
-							'name' => $object['name'],
+							'name' => $classification === CalDavBackend::CLASSIFICATION_CONFIDENTIAL && $user !== $owner ? 'Busy' : $object['name'],
```
Comment (tcitworld) [inline apps/dav/lib/CalDAV/Activity/Backend.php]: How is `Busy` used afterwards? Shouldn't it be translated?

Comment (rullzer) [reply apps/dav/lib/CalDAV/Activity/Backend.php]: It is translated: https://github.com/nextcloud/server/pull/13430/files#diff-655d608107af51062fbdcce130f97d1eR176

Comment (nickvergessen) [reply apps/dav/lib/CalDAV/Activity/Backend.php]: This is the value that goes to the database, and is translated when the activity is rendered later on.

**Pull request merged**